CI/install_libiio.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+sudo apt-get -y update
+sudo apt-get -y install git build-essential libxml2-dev bison flex libcdk5-dev cmake libaio-dev libusb-1.0-0-dev libserialport-dev libavahi-client-dev doxygen graphviz python3 python3-pip python3-setuptools
+
+git clone -b libiio-v0 https://github.com/analogdevicesinc/libiio.git
```

Comment:
Maybe would be better to have libiio branch in a variable. We'll change easier back to 'main' whenever there will be the case.